docs/cni/kubernetes/Policy.md

```diff
@@ -0,0 +1,133 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Policy for Kubernetes
+The Calico CNI plugin for Kubernetes allows you to specify network policy in the Kubernetes API using annotations.  
+> *Note*: annotation-based policy is currently experimental and is subject to change in future releases. 
+
+## Prerequisites
+* A Kubernetes v1.1 Deployment using the Calico CNI plugin.
+* You must be using the iptables kube-proxy in your deployment. This is the default proxy mode in Kubernetes v1.1.3.
```

Comment:
Having come from https://github.com/projectcalico/calico-containers/blob/cni-docs-integration/docs/cni/kubernetes/VagrantCoreOS.md, how do I know what I've got set up?

Either the VagrantCoreOS,md should tell me what I've set up, or this doc should explain how I would tell.